tables.go

```diff
@@ -0,0 +1,214 @@
+package fastjson
+
+import "unicode/utf8"
+
+// safeSet holds the value true if the ASCII character with the given array
```

Comment:
Should we remove `safeSet` if it's not used? We can always add it later.

Reply:
Done!